CMakeSettings.json

```diff
@@ -1,5 +1,41 @@
 ﻿{
   "configurations": [
+    {
+      "name": "x64-DebugARM64",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_arm64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
+      "installRoot": "${projectDir}\\out\\install\\${name}",
+      "cmakeCommandArgs": "-DINSTALL_GTEST=OFF -DBUILD_TESTING=OFF -DBUILD_TRANSPORT_CURL=OFF -DBUILD_SAMPLES=ON -DBUILD_TRANSPORT_WINHTTP=ON -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION=10.0",
+      "buildCommandArgs": "-v",
+      "ctestCommandArgs": "",
+      "variables": [
+        {
+          "name": "VCPKG_TARGET_TRIPLET",
+          "value": "arm64-WindowsStore-static",
```

Comment:
```suggestion
          "value": "arm64-uwp",
```

Reply:
[based on https://vcpkg.readthedocs.io/en/latest/users/triplets/ the -uwp triplets are "community" and not officially supported ( not tested by CI builds) the officially supported thing is WindowsStore

Reply:
WindowsStore is may be the value for `VCPKG_CMAKE_SYSTEM_NAME`, not `VCPKG_TARGET_TRIPLET`. Vcpkg triplet is taken from https://github.com/microsoft/vcpkg/tree/master/triplets.
`arm-uwp` is official, `arm64-uwp` is community. We can compile on both.

Reply:
And `arm64-uwp` is community for vcpkg. And SDK successfully compile there. We don't have to match vcpkg in the supported platforms exactly.